generated_api_shadow/envoy/config/retry/previous_hosts/v3/BUILD

```diff
@@ -0,0 +1,9 @@
+# DO NOT EDIT. This file is generated by tools/proto_sync.py.
```

Comment:
I tried to run this in my local, and it prints: 

```
# DO NOT EDIT. This file is generated by tools/proto_format/proto_sync.py.
```

Reply:
humm. can we discuss in slack quickly to understand it more ?

Reply:
strange !!!! 

 it is also same as in my env as `# DO NOT EDIT. This file is generated by tools/proto_format/proto_sync.py. `

one also quick thing. how it can be different wrt tmaster as i have not made nay update in `./tools/proto_format/proto_format.sh` and also my local is sync with master. :)

Reply:
It is not "strange !!!!" :).

You just need to:

1. Merge (upstream) master.
2. Run `./tools/proto_format/proto_format.sh` fix again

```
$ git clone git@github.com:ankatare/envoy.git
$ cd envoy
$ git remote add upstream git@github.com:envoyproxy/envoy.git
$ git fetch upstream
$ git checkout retry_previous_hosts
$ git merge upstream/master
$ ./tools/proto_format/proto_format.sh fix^
$ git diff
diff --git a/generated_api_shadow/envoy/config/retry/previous_hosts/v3/BUILD b/generated_api_shadow/envoy/config/retry/previous_hosts/v3/BUILD
index ef3541ebc..ee92fb652 100644
--- a/generated_api_shadow/envoy/config/retry/previous_hosts/v3/BUILD
+++ b/generated_api_shadow/envoy/config/retry/previous_hosts/v3/BUILD
@@ -1,4 +1,4 @@
-# DO NOT EDIT. This file is generated by tools/proto_sync.py.
+# DO NOT EDIT. This file is generated by tools/proto_format/proto_sync.py.
 
 load("@envoy_api//bazel:api_build_system.bzl", "api_proto_package")
```

Reply:
ok. let me try this. thanks a lot for your quick reply :)